akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerSpec.scala

```diff
@@ -107,6 +107,7 @@ class CircuitBreakerSpec extends AkkaSpec with BeforeAndAfter {
       val breaker = CircuitBreakerSpec.shortResetTimeoutCb()
       intercept[TestException] { breaker().withSyncCircuitBreaker(throwException) }
       checkLatch(breaker.halfOpenLatch)
+      breaker.openLatch.reset
```

Comment:
Good.
The exception above will put it in Open followed by HalfOpen, and below we want to check that it transitions to Open again